Main.hs

```diff
@@ -50,38 +54,38 @@ basilica origin db emailChan = scottyApp $ do
     withPost json =<< param "id"
   get "/posts" $ do
     since <- maybeParam "after"
```

Comment:
The lines below (because github will split them up) demonstrate a downside, but a temporary one -- all my `liftIO`s had to become `liftDB db`s. However, a future refactoring is going to completely split the `Scotty` stuff from the `Database` stuff so that this code will exist in one place instead of in N places.